es-vue-base/README.md

```diff
@@ -6,6 +6,20 @@ Initial layout was set-up following <https://github.com/team-innovation/vue-sfc-
 
 - `es-bs-base` is a peer-dependency for CSS styling
 
+### Peer Dependencies
+
+`es-vue-base` requires the following peer-dependencies:
+
+- `vue`
+- `bootstrap-vue`
+- `html-truncate`
+- `vue-slider-component`
+- `vuelidate`
```

Comment:
Based on our conversation + my testing I'm going to eliminate these three as requirements